safe-global/web-core
# feat: transaction guard removal

## What it solves

Resolves transaction guard removal

## How this PR fixes it

The option to remove transaction guards via a removal modal has now been implemented into the settings.

## How to test it

Enable a transaction guard (DenyListGuard: `0x885dCFA4A720dBAEa1Dd3AB1bB7c31Ed5f04B1c6`) then remove it via the settings. The removal should succeed after execution and the guard no longer be active/visible in the settings.

## Analytics changes

Removal tracks a new "Remove transaction guard" event.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/190403945-00980e99-e64c-48d6-946a-962c6c91c219.png)

![image](https://user-images.githubusercontent.com/20442784/190404052-6a3f5225-c2a1-4dce-a0fb-7c8590ecd7a5.png)

![remove guard](https://user-images.githubusercontent.com/20442784/191742209-accf11f2-168b-42bc-97a5-2e9832f86575.gif)
